src/TraceEvent/Utilities/command.cs

```diff
@@ -422,16 +422,14 @@ public Command(string commandLine, CommandOptions options)
             if (options.elevate)
             {
                 options.useShellExecute = true;
-                startInfo.Verb = "runas";
+                startInfo.Arguments = "runas " + startInfo.Arguments;
```

Comment:
Have we tested this?

Reply:
I hadn't. I thought I knew what it was doing after reading the docs, but I should have, because this doesn't work. I am working on a alternate solution.

Reply:
I've been looking for a solution to this and so far nothing great. Furthermore, it seems that `UseShellExecute` is actually [not supported on core 1.1](https://github.com/dotnet/corefx/blob/release/1.1.0/src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs#L42-L49)

Seeing as this code is only concerned with Windows anyway it would be possible to solve this by adopting a portion of the source from corefx 2.0, but I haven't determined just how big that chunk would need to be yet

I'll keep looking for another solution in the meantime

Reply:
The prospect of supporting this in netstandard1.6 seems to be surprisingly difficult. My current thought is to throw `PlatformNotSupportedException` on netstandard1.6 through the use of an #if condition. That way we can still support perfview working just as before, but leave this unimplemented in netstandard1.6. If this is the approach then it seems reasonable to additionally target netstandard2.0 so users in core 2.0 on Windows can use this function

Reply:
Yes, as mentioned above, making a new target for 1.6 and #ifdefs seems and throwing when needed is very reasonable.   Do that.